content/projects/gsoc/2023/project-ideas/plugin-installation-manager-tool.adoc

```diff
@@ -31,6 +31,10 @@ students are invited to introduce new features and improvements in the plugin in
 The list of enhancements is a subject for discovery as a part of the application process.
 Some of the ideas are available in the link:https://github.com/jenkinsci/plugin-installation-manager-tool/issues[project tracker].
 
+Additional background information may be found in a video recording from 2021.
+
+video::QJcwcLnHjRw[youtube]
```

Comment:
@MarkEWaite I think for some reason the changes made to the dimensions of the YouTube video was not picked up.

Reply:
Thanks.  I missed the commit.  Committed in a92c2c699a131db3cfc9af3ba73a64c6fbfc3bb4